src/index.js

```diff
@@ -10,19 +10,19 @@ import {
 	differenceInMonths, differenceInQuarters, differenceInYears,
 	endOfSecond, endOfMinute, endOfHour, endOfDay,
 	endOfWeek, endOfMonth, endOfQuarter, endOfYear
-} from 'date-fns';
+} from 'date-fns/esm';
 
-var FORMATS = {
-	datetime: 'MMM D, YYYY, h:mm:ss a',
-	millisecond: 'h:mm:ss.SSS a',
-	second: 'h:mm:ss a',
-	minute: 'h:mm a',
+const FORMATS = {
+	datetime: 'MMM d, yyyy, h:mm:ss aaaa',
+	millisecond: 'h:mm:ss.SSS aaaa',
+	second: 'h:mm:ss aaaa',
+	minute: 'h:mm aaaa',
 	hour: 'ha',
```

Comment:
`haaaa`?

Reply:
"ha" format is fine. Reference for the format token conversion was the (core) moment.js adapter: eg 10AM (capital AP/PM)